src/Dapr.Extensions.Configuration/DaprSecretStoreConfigurationSource.cs

```diff
@@ -18,7 +18,7 @@ public class DaprSecretStoreConfigurationSource : IConfigurationSource
         /// <summary>
         /// Gets or sets the store name.
         /// </summary>
-        public string Store { get; set; }
+        public string Store { get; set; } = string.Empty;
```

Comment:
This should also be initialized with `default!`

Reply:
Changed the default value but I wonder if an empty string would not be better than making it "secretly" null.

Reply:
Part of it is personal preference (based on experience). Using `null` as a sentinel value is a far better practice than `string.Empty`. You actually **want** to get an exception when you access an uninitialized value.

The challenge here is that it's hard to retrofit nullability onto existing APIs. The design of this class is not a good fit, but unfortunately can't be easily changed.